pyproject.toml

```diff
@@ -1,25 +1,272 @@
+[build-system]
+requires = ["hatchling", "hatch-vcs"]
+build-backend = "hatchling.build"
+
+
 [project]
-name = "open_pyfus"
-version = "0.1.0"
+name = "openlifu"
+authors = [
+  { name = "Peter Hollender", email = "peterhollender@gmail.com" },
+]
 description = "Openwater Focused Ultrasound Toolkit"
+readme = "README.rst"
+license.file = "LICENSE"
+requires-python = ">=3.8"
+classifiers = [
+  "Development Status :: 1 - Planning",
+  "Intended Audience :: Science/Research",
+  "Intended Audience :: Developers",
+  "License :: OSI Approved :: MIT License",
+  "Operating System :: OS Independent",
+  "Programming Language :: Python",
+  "Programming Language :: Python :: 3",
+  "Programming Language :: Python :: 3 :: Only",
+#  "Programming Language :: Python :: 3.8", # TODO: Uncomment whatever of these python versions we find to be okay.
+#  "Programming Language :: Python :: 3.9",
+#  "Programming Language :: Python :: 3.10",
+  "Programming Language :: Python :: 3.11",
+  "Programming Language :: Python :: 3.12",
+  "Topic :: Scientific/Engineering",
+  "Typing :: Typed",
+]
+dynamic = ["version"]
 dependencies = [
-    "xarray",
-    "numpy",
-    "matplotlib",
-    "pandas",
-    "scipy",
-    "vtk",
-    "h5py",
-    "opencv-contrib-python",
-    "crc",
-    "nibabel",
-    "sphinx",
-    "ipykernel"
-]
-
-[tool.setuptools]
-include-package-data = false
```

Comment:
@peterhollender I removed this setting `include-package-data = false`, was there a reason to have it? Just checking in case I missed something